### PR TITLE
[wasm][debugger] Improve array visualization

### PR DESCRIPTION
**Pull request by radical**
TODO: viz for multidim arrays

### CODE REVIEW

File: sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs [hunk 1]
```diff
@@ -206,7 +206,25 @@ protected override async Task<bool> AcceptCommand (MessageId id, string method,
...
+								if (!IsValueTypeCached (id, objId)) {
+									var arrayObjectId = int.Parse (parts [2]);
+									var idx = int.Parse (parts [3]);
+									await GetDetails (id, MonoCommands.GetArrayValueExpanded (arrayObjectId, idx), token, send_response: false);
+								}
+								GetDetailsForValueType (id, objId, token);
+								break;
+							}
+							default:
+								throw new ArgumentException ($"Unknown objectId format for array: {objId}");
```
Comment (lewing) [inline sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs:225]: We don't really want the proxy throwing uncaught exceptions.  `Runtime.getProperties` has exceptionDetails in the result so let's fill that in instead.   Also please move this out of the switch and into a function.

**Comment by lewing** [comment]
@monojenkins build failed

**Comment by radical** [comment]
@monojenkins build failed

**Comment by lewing** [comment]
@monojenkins build Linux ARMv7

**Comment by lewing** [comment]
@monojenkins build Linux ARMv7

**Pull request merged**